### ISSUE TITLE
multi-camera-multi-object-detection with re-identification

### ISSUE THREAD

**Issue by muhammad-usama-aleem** [NirAharon/BoT-SORT#94]
how can we track the same person person with different cameras? 

For example, if a person is detected by a camera, then this person is labeled with an id number, then this person can keep this id number to the other camera.

**Comment by jarvijd** [comment]
just wondering - did you ever find an answer to this question?

**Comment by muhammad-usama-aleem** [comment]
no, as you can see. I found some repositories which does not work now and even there dataset is not available. let me know if you get any lead.

**Comment by rolson24** [comment]
My guess is that you would want to make a model that is similar to the Fast ReID model (but more powerful and accurate) to get the embedding vectors for each detection box, and then match the detections from one camera to the next based on how close they are in the embedding space. Honestly it sounds like you would need some sort of facial recognition or something depending on if the cameras are looking at the same area or not, and how long you want the tracker to remember a person.
